expo-zustand-styled-components/src/interface/repositories.interface.ts

```diff
@@ -0,0 +1,21 @@
+export interface Repository {
+  id: number;
+  name: string;
+  full_name: string;
+  owner: { login: string; type: 'User' | 'Organization' };
+  description: string;
+  private: boolean;
+  html_url: string;
+  url: string;
+  updated_at: Date;
+  stargazers_count: number;
+  language: string;
+  branches_url: string;
+  visibility: 'public' | 'private';
+  subscribers_count: number;
```

Comment:
We have a types folder for declaring interfaces, and this might have been done.

Reply:
Are you sure? As of when I did this, there was nothing like that.